### DISCUSSION TITLE
貌似解决了1028的问题了

### DISCUSSION THREAD

**Discussion by Aellyt** [Ljzd-PRO/Mys_Goods_Tool#151] [Show and tell]
![68DC809B-6804-4DE2-90C9-B4AA070D5BD8](https://github.com/Ljzd-PRO/Mys_Goods_Tool/assets/83526546/417f880d-3464-492e-ae4f-084caa05cab8)
今天中午跑的，抢到了

**Comment by Ljzd-PRO** [comment]
应该也是比较看运气

**Reply by Aellyt** [reply]
并不是运气，请求的参数少了点，’Content-Type' : `application/json;charset=utf-8`大佬你可以看下请求的是不是少了后面的charset=utf-8

**Comment by kuronekoda** [comment]
我昨天抢还是返回1028，抢的是星穹铁道流沙周边

**Comment by qwqzz** [comment]
这次第4秒换到了60原石，150没换到，客户端跳验证码了
